src/python_pybind11/test/SphericalCoordinates_TEST.py

```diff
@@ -0,0 +1,570 @@
+# Copyright (C) 2021 Open Source Robotics Foundation
```

Comment:
Should we move the file instead of copying it? It's easier to see if there's a diff from the existing test